apps/frontend/components/TableDisplay/TableDisplay.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable max-lines */
```

Comment:
`formatForms` and/or `TableHeader` could live in different file, to remove this exception